### PR TITLE
[daint-gpu] Octave and dependencies for CrayGNU-18.08

### PR DESCRIPTION
**Pull request by omlins**

### CODE REVIEW

**Comment by omlins** [comment]
retest this please

File: easybuild/easyconfigs/o/Octave/Octave-5.1.0-CrayGNU-18.08.eb [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+description = "GNU Octave is a high-level interpreted language, primarily intended for numerical computations."
+
+toolchain = {'name': 'CrayGNU', 'version': '18.08'}
+toolchainopts = {'pic': True, 'dynamic': True}
+
+source_urls = [GNU_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+
+builddependencies = [
+    ('Bison', '3.3.2'),
```
Comment (lucamar) [inline easybuild/easyconfigs/o/Octave/Octave-5.1.0-CrayGNU-18.08.eb]: @omlins What about old dependencies that I have found in the previous recipes? E.g. `texinfo`

**Comment by omlins** [comment]
retest this please

File: easybuild/easyconfigs/g/GL2PS/GL2PS-1.3.9-CrayGNU-18.08.eb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+homepage = 'http://www.geuz.org/gl2ps/'
+description = "GL2PS: an OpenGL to PostScript printing library"
+
+toolchain = {'name': 'CrayGNU', 'version': '18.08'}
+
+source_urls = ['http://geuz.org/%(namelower)s/src/']
+sources = [SOURCELOWER_TGZ]
+
+builddependencies = [
+    ('CMake', '3.6.0'),
```
Comment (lucamar) [inline easybuild/easyconfigs/g/GL2PS/GL2PS-1.3.9-CrayGNU-18.08.eb]:
```suggestion
    ('CMake', '3.12.4'),
```

File: easybuild/easyconfigs/l/LLVM/LLVM-3.8.1-CrayGNU-18.08.eb [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+ to use LLVM as an optimizer and code generator."""
+
+toolchain = {'name': 'CrayGNU', 'version': '18.08'}
+toolchainopts = {'cstd': 'gnu++11'}
+
+source_urls = ['http://%(namelower)s.org/releases/%(version)s']
+sources = ['%(namelower)s-%(version)s.src.tar.xz']
+
+builddependencies = [
+    ('CMake', '3.6.0'),
```
Comment (lucamar) [inline easybuild/easyconfigs/l/LLVM/LLVM-3.8.1-CrayGNU-18.08.eb]:
```suggestion
    ('CMake', '3.12.4'),
```

File: easybuild/easyconfigs/g/GLPK/GLPK-4.60-CrayGNU-18.08.eb [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+ It is a set of routines written in ANSI C 
+ and organized in the form of a callable library."""
+
+toolchain = {'name': 'CrayGNU', 'version': '18.08'}
+
+source_urls = ['http://ftp.gnu.org/gnu/%(namelower)s/']
+sources = [SOURCELOWER_TAR_GZ]
+
+dependencies = [
+    ('GMP', '6.1.1'),
```
Comment (lucamar) [inline easybuild/easyconfigs/g/GLPK/GLPK-4.60-CrayGNU-18.08.eb]:
```suggestion
    ('GMP', '6.1.2'),
```

File: easybuild/easyconfigs/n/nettle/nettle-3.2-CrayGNU-18.08.eb [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+ languages (C++, Python, Pike, ...), in applications like LSH or GNUPG,
+ or even in kernel space."""
+
+toolchain = {'name': 'CrayGNU', 'version': '18.08'}
+
+source_urls = [GNU_SOURCE]
+sources = [SOURCE_TAR_GZ]
+
+dependencies = [
+    ('GMP', '6.1.1'),
```
Comment (lucamar) [inline easybuild/easyconfigs/n/nettle/nettle-3.2-CrayGNU-18.08.eb]:
```suggestion
    ('GMP', '6.1.2'),
```

File: easybuild/easyconfigs/o/Octave/Octave-5.1.0-CrayGNU-18.08.eb [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+    ('flex', '2.6.4'),
+    ('Autotools', '20180311'),
+    ('libtool', '2.4.6'),
+    ('gperf', '3.0.4'),
```
Comment (lucamar) [inline easybuild/easyconfigs/o/Octave/Octave-5.1.0-CrayGNU-18.08.eb]:
```suggestion
    ('gperf', '3.1'),
```

File: easybuild/easyconfigs/g/gperf/gperf-3.0.4-CrayGNU-18.08.eb [hunk 7]
```diff
@@ -0,0 +1,23 @@
+easyblock = 'ConfigureMake'
+
+name = 'gperf'
+version = '3.0.4'
```
Comment (lucamar) [inline easybuild/easyconfigs/g/gperf/gperf-3.0.4-CrayGNU-18.08.eb]:
```suggestion
version = '3.1'
```

File: easybuild/easyconfigs/c/cURL/cURL-7.49.1-CrayGNU-18.08.eb [hunk 8]
```diff
@@ -0,0 +1,30 @@
+easyblock = 'ConfigureMake'
+
+name = 'cURL'
+version = '7.49.1'
```
Comment (lucamar) [inline easybuild/easyconfigs/c/cURL/cURL-7.49.1-CrayGNU-18.08.eb]:
```suggestion
version = '7.61.1'
```

File: easybuild/easyconfigs/o/Octave/Octave-5.1.0-CrayGNU-18.08.eb [hunk 9]
```diff
@@ -0,0 +1,58 @@
...
+    ('PCRE', '8.38'),
+    ('ncurses', '6.1'),
+    ('arpack-ng', '3.3.0'),
+    ('cURL', '7.49.1'),
```
Comment (lucamar) [inline easybuild/easyconfigs/o/Octave/Octave-5.1.0-CrayGNU-18.08.eb]:
```suggestion
    ('cURL', '7.61.1'),
```

**Comment by omlins** [comment]
http://www.nasm.us/ is down. This might be temporary.

**Comment by omlins** [comment]
retest this please

**Comment by omlins** [comment]
retest this please

**Comment by omlins** [comment]
retest this please

**Comment by omlins** [comment]
retest this please

**Pull request closed without merging**